src/main/java/io/rebloom/client/ClusterClient.java

```diff
@@ -234,7 +254,134 @@ public Boolean execute(Jedis connection) {
         }).run(name);
     }
 
+    /**
+     * TOPK.RESERVE key topk width depth decay
+     *
+     * Reserve a topk filter.
+     * @param key The key of the filter
+     * @param topk
+     * @param width
+     * @param depth
+     * @param decay
+     *
+     * Note that if a filter is not reserved, a new one is created when {@link #add(String, byte[])}
+     * is called.
+     */
+    public void topkCreateFilter(String key, long topk, long width, long depth, double decay) {
+        (new JedisClusterCommand<Void>(this.connectionHandler, this.maxAttempts){
+            @Override
+            public Void execute(Jedis jedis) {
+                Connection conn = jedis.getClient();
+                conn.sendCommand(TopKCommand.RESERVE, SafeEncoder.encode(key), Protocol.toByteArray(topk),
+                        Protocol.toByteArray(width), Protocol.toByteArray(depth),Protocol.toByteArray(decay));
+                String resp = conn.getStatusCodeReply();
+                if (!resp.equals("OK")){
+                    throw new JedisException(resp);
+                }
+                return null;
```

Comment:
why return null?

Reply:
because type of  Void need to return null.

Reply:
I know, but it will create backward issue for existing users